mibs/cdata/SNMPv2-MIB

```diff
@@ -0,0 +1,773 @@
+-- Changes to rfc1907 - SNMPv2-MIB (object for SNMPv2 protocol)
+--      NOTE: the following should be in a group -

```

Comment:
Please remove any standard mib file (that is already in ./mibs/) from this directory. Only vendor specific mibs should be here.
If a standard mib is missing from ./mibs/, then please add it there instead od the vendor specific directory